oe_paragraphs.install

```diff
@@ -254,3 +255,36 @@ function oe_paragraphs_update_8004(&$sandbox) {
   }
   $form_display->save();
 }
+
+/**
+ * Updates the allowed_values management of field_oe_icon.
+ */
+function oe_paragraphs_update_8005(&$sandbox) {
```

Comment:
we'r not doing schema changes so this should go into a post_update, oe_paragraphs_post_update_10007, see https://github.com/openeuropa/oe_paragraphs/blob/master/oe_paragraphs.post_update.php#L188